src/js/disability-benefits/526EZ/actions/index.js

```diff
@@ -0,0 +1,64 @@
+import Raven from 'raven-js';
+/* eslint-disable */
+import { apiRequest } from '../../../../platform/utilities/api';
+import { fetchInProgressForm } from '../../../common/schemaform/save-in-progress/actions';
+
+export const ITFStatuses = Object.freeze({
+  active: 'active',
+  claim_received: 'claim_received', // eslint-disable-line camelcase
+  duplicate: 'duplicate',
+  expired: 'expired',
+  pending: 'pending',
+  incomplete: 'incomplete'
+});
+
+const delay = (ms) => new Promise(resolve =>
+  setTimeout(resolve, ms)
+);
+
+export function submitIntentToFile(formConfig, onChange) {
+  const { intentToFileUrl, formId, migrations, prefillTransformer } = formConfig;
+  let ITFStatus = ITFStatuses.pending;
+  return dispatch => {
+
+    onChange({ ITFStatus });
+
+    return delay(2000).then(() => {
+      ITFStatus = ITFStatuses.active;
+      onChange({ ITFStatus });
+
+      // TODO: if the backend handles resubmission, this check can be removed
+      return ITFStatus === 'active';
+    })
+      .catch(() => {
+        const errorMessage = 'Network request failed';
```

Comment:
@peggygannon Before we can start the form, we have to make a request for an ITF (intent to file). If that call fails, we can't start the form. What error message should we show to the user if that happens?